wklink/LICENSE

```diff
@@ -0,0 +1,21 @@
+MIT License
```

Comment:
Should we just use an MIT License for the whole repo? Or might this create conflicts with the software we use in the image?

Reply:
Good question. We can put the dockerfiles + config under MIT, but keep the original license of the software

Reply:
Not sure how to word that, tho. Let's delay to a different PR.

Reply:
I moved the license up. Helm does it the same way. They also have a separate license for the charts.